service/src/main/java/bio/terra/pipelines/common/utils/JobStatusEnum.java

```diff
@@ -0,0 +1,7 @@
+package bio.terra.pipelines.common.utils;
+
+public enum JobStatusEnum {
```

Comment:
I imagine in future adding more descriptive statuses like `COPYING INPUT DATA` and `RUNNING IMPUTATION PIPELINE` but we can discuss.

likely will be renamed to something more descriptive like `TspsJobStatusEnum` when we do the async API refactoring

Reply:
Yea so i think these should be statuses of what step the pipeline might be on.  Maybe having a parent class of `CommenJobStatusEnum` and then a subclass of `ImputationPipelineJobStatusEnum`.